src/sentry/static/sentry/app/actionCreators/globalSelection.tsx

```diff
@@ -113,22 +109,26 @@ export function initializeUrlState({
   const orgSlug = organization.slug;
   const query = pick(queryParams, [URL_PARAM.PROJECT, URL_PARAM.ENVIRONMENT]);
   const hasProjectOrEnvironmentInUrl = Object.keys(query).length > 0;
-  const parsed = getStateFromQuery(queryParams, {allowAbsoluteDatetime: showAbsolute});
+  const parsed = getStateFromQuery(queryParams, {
+    allowAbsoluteDatetime: showAbsolute,
+    allowEmptyPeriod: true,
+  });
 
-  let globalSelection: Omit<GlobalSelection, 'datetime'> & {
-    datetime: {
-      [K in keyof GlobalSelection['datetime']]: GlobalSelection['datetime'][K] | null;
-    };
-  } = {
+  let globalSelection = {
     ...getDefaultSelection(),
-    datetime: {
-      [DATE_TIME.START as 'start']: parsed.start || null,
-      [DATE_TIME.END as 'end']: parsed.end || null,
-      [DATE_TIME.PERIOD as 'period']: parsed.period || null,
-      [DATE_TIME.UTC as 'utc']: parsed.utc || null,
-    },
```

Comment:
Ah I see, this merging was wrong... I would do something like, destructure `datetime` from `defaultSelection` if it exists and append it like it currently is. That way the precedence is little more clear and in one place:

```
const {datetime: customizedDefaultDateTime, ...customizedDefaultSelection} = defaultSelection || {};

...

...getDefaultSelection(),
...customizedDefaultSelection,
    datetime: {	
      [DATE_TIME.START as 'start']: parsed.start || customizedDefaultDateTime.start || null,	
      [DATE_TIME.END as 'end']: parsed.end || customizedDefaultDateTime.end  || null,	
      [DATE_TIME.PERIOD as 'period']: parsed.period || customizedDefaultDateTime.period  || null,	
      [DATE_TIME.UTC as 'utc']: parsed.utc || customizedDefaultDateTime.utc  || null,	
    },

```